sdks/python/apache_beam/transforms/cy_combiners.py

```diff
@@ -22,6 +22,33 @@
 from apache_beam.transforms import core
 
 
+__all__ = [
+    'AccumulatorCombineFn',
+    'CountAccumulator',
```

Comment:
I don't think any of these accumulator types should be public. Actually, to be safe, just make this entire module private.